test/CMakeLists.txt

```diff
@@ -4,6 +4,7 @@
 # Copyright (c) 2021 Vaibhav Thakkar
```

Comment:
also be sure to try your code with MKL

Reply:
I used MKL and the timing results in this module are very similar (somewhat smaller in the larger example). I will put assertions in the benchmark that correspond to the maximum of both preparation times.

Reply:
sounds good to me